GliaWidgets/Public/Glia/Glia+RestoreEngagement.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+extension Glia {
+    func restoreOngoingEngagement(
+        configuration: Configuration,
+        currentEngagement: CoreSdkClient.Engagement,
+        maximize: Bool
+    ) {
+        // Creates interactor instance
+        let createdInteractor = setupInteractor(
+            configuration: configuration,
+            queueIds: []
+        )
+
+        // In case engagement should be restarted, LO ack should not
+        // be appeared again.
+        createdInteractor.skipLiveObservationConfirmations = true // TODO: snack bar has to be shown, but not dialog
+
+        // Apply company name to theme and get the modified theme.
+        let modifiedTheme = applyCompanyName(using: configuration, theme: theme)
+
+        let viewFactory = ViewFactory(
+            with: modifiedTheme,
+            messageRenderer: messageRenderer,
+            environment: .init(
```

Comment:
Can environment be initialized in a static method within `viewFactory` environment's extension?

Reply:
Updated to use it 07047ccf370e05a605a583b15d64237dd3d6e307.

Reply:
Addressed this in #996 via a02dcc8b22d9355ca40af7e846e873106bf90f18.